libs/packages/sam-formly/src/lib/formly/types/input.ts

```diff
@@ -3,19 +3,33 @@ import { FieldType } from '@ngx-formly/core';
 
 @Component({
   selector: 'sds-formly-field-input',
-  template: `
+  template: ` <div class="maxw-mobile-lg position-relative">
     <input
       [ngClass]="{
         'usa-input--disabled': to.disabled,
         'usa-input--error': showError
       }"
-      class="usa-input"
+      class="usa-input padding-right-3"
       [placeholder]="to.placeholder"
       name="input-success"
       [formlyAttributes]="field"
       [type]="to.inputType ? to.inputType : 'text'"
       [formControl]="formControl"
     />
-  `,
+    <ng-container *ngIf="field.formControl.value">
+      <span class="position-absolute right-105 top-1 cursor-pointer bg-white">
+        <span
```

Comment:
Any reason to not just use an unstyled button here? That way you don't have to mess with the click and keyup events or the tabindex.